### ISSUE TITLE
More combinatorial sequences (wishlist)

### ISSUE THREAD

**Issue by videlec** [fredrik-johansson/arb#261]
[K. Brigmann](http://www.mi.uni-koeln.de/Bringmann) developed analogues of Rademacher formula for many other integral sequences. It is tempting to write certified numerical algorithms for evaluating them.

Ideally, there should exist a compiler that would turn modularity properties of generating function into a certified algorithm for numerical evaluation of the coefficients.

**Comment by fredrik-johansson** [comment]
I agree, I had the idea for such compiler years ago but the details are beyond me :)
